### PR TITLE
Add a build prompt to the UI

### PR DESCRIPTION
**Pull request by blink1073**
Partially addresses #2244 by telling the user that they need to manually rebuild from the front end.  

<image src="https://cloud.githubusercontent.com/assets/2096628/26449084/ec8ad550-4115-11e7-8d2f-c7925fce6ecc.png" width=300>

### CODE REVIEW

**Comment by ellisonbg** [comment]
Looks like this doesn't have a UI yet though?

**Comment by blink1073** [comment]
Ha, hence the WIP :wink:

**Comment by ellisonbg** [comment]
Feel free to ping @cameronoelsen for any design help.

**Comment by blink1073** [comment]
Updated to include a screenshot of the startup message.

**Comment by blink1073** [comment]
Put it back to WIP until I add tests for the new function.

File: packages/application-extension/src/index.ts [hunk 1]
```diff
@@ -87,6 +91,22 @@ const plugin: JupyterLabPlugin<void> = {
     });
     palette.addItem({ command, category });
 
+    // Temporary build message for manual rebuild.
+    let buildMessage = PageConfig.getOption('buildRequired');
+    if (buildMessage) {
+      let body = document.createElement('pre');
```
Comment (ellisonbg) [inline packages/application-extension/src/index.ts]: Let's make this a regular p tag so the font isn't monospaced.

Comment (ellisonbg) [inline packages/application-extension/src/index.ts]: Although, maybe put the `jupyter lab build` in a pre tag.

**Comment by blink1073** [comment]
Updated with new screenshot.

**Comment by ellisonbg** [comment]
Merging!

**Pull request merged**